theseus/extlib/test_cusolver_lu_solver.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import pytest  # noqa: F401
+import torch  # needed for import of Torch C++ extensions to work
+from scipy.sparse import csr_matrix
+
+from theseus.extlib.sparse_matrix_utils import generate_sparse_matrix
+
+
+def check_lu_solver(base_batch_size, batch_size, rows, cols, fill, verbose=False):
```

Comment:
We should have this file inside `extlib/tests/` dir following convention and for later as this module grows.

Reply:
doing so in next rev.